packages/devtools_app/lib/src/screens/inspector/inspector_screen.dart

```diff
@@ -580,10 +580,7 @@ class InspectorDefaultDetailsViewOption extends StatelessWidget {
         return Column(
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
-            Text(
-              'Select the default tab for the inspector.',
-              style: theme.subtleTextStyle,
-            ),
+            const Text('Select the default tab for the inspector.'),
```

Comment:
manually apply theme.regularTextStyle to this text (I'm working on another PR where not having the style  will cause theming bugs)